### PR TITLE
Jenkins re-enable Catkin and Colcon builds

### PR DESCRIPTION
**Pull request by dagar**
- bump tag to 2019-02-13

### CODE REVIEW

**Comment by TSC21** [comment]
@dagar The only blocker here was the ROS Mavlink version inside the container. This probably requires a new container update so to bring the new ROS Mavlink version to CI.

File: Jenkinsfile [hunk 1]
```diff
@@ -8,72 +8,72 @@ pipeline {
...
+            docker {
+              image 'px4io/px4-dev-ros2-bouncy:2019-03-08'
+              args '-e CCACHE_BASEDIR=$WORKSPACE -v ${CCACHE_DIR}:${CCACHE_DIR}:rw -e HOME=$WORKSPACE'
+            }
+          }
+          steps {
+            sh 'ls -l'
+            sh '''#!/bin/bash -l
+              echo $0;
+              unset ROS_DISTRO;
```
Comment (dagar) [inline Jenkinsfile]: Why is this needed?

Comment (TSC21) [reply Jenkinsfile]: On a environment with multiple ROS versions installed, if one previously have sourced a certain ROS distro env setup, different from the one you are sourcing now, there will be a warning message similar to `ROS_DISTRO was set to 'crystal' before. Please make sure that the environment does not mix paths from different distributions.`. This just avoids that this message spams the console, as we first unset the `ROS_DISTRO` env variable so there are no conflicts.
